web/packages/teleport/src/components/UserMenuNav/UserMenuNav.tsx

```diff
@@ -257,21 +258,29 @@ export function UserMenuNav({ username }: UserMenuNavProps) {
 
         <DropdownDivider />
 
-        <DropdownItem
-          open={open}
-          style={{
-            transitionDelay: `${transitionDelay}ms`,
-          }}
-        >
-          <DropdownItemButton onClick={onThemeChange}>
-            <DropdownItemIcon>
-              {preferences.theme === ThemePreference.Light ? <Sun /> : <Moon />}
-            </DropdownItemIcon>
-            Switch to{' '}
-            {preferences.theme === ThemePreference.Dark ? 'Light' : 'Dark'}{' '}
-            Theme
-          </DropdownItemButton>
-        </DropdownItem>
+        {/* Hide ability to switch themes if custom theme is set. */}
+        {!(cfg.customTheme === 'bblp') && (
```

Comment:
I guess we can just check for `!cfg.customTheme`.

Reply:
Yeah, let's refer to `'bblp'` as little as possible.